lua/neo-tree/git/ignored.lua

```diff
@@ -74,13 +65,22 @@ M.mark_ignored = function(state, items)
       result = {}
     end
 
+    -- on Windows, git seems to return quotes and double backslash "path\\directory"
```

Comment:
Since this part is Windows specific, can you wrap it in an `if utils.is_windows` block?

Reply:
Done 👍